modules/pytket-braket/tests/backend_test.py

```diff
@@ -377,3 +377,6 @@ def test_postprocess_ionq() -> None:
 def test_retrieve_available_devices() -> None:
     backend_infos = BraketBackend.available_devices()
     assert len(backend_infos) > 0
+    # Test annealers are filtered out.
```

Comment:
Does this test for that?

Reply:
Currently, all D-wave devices are hosted in us-west-2. This test should cover it as long as AWS doesn't change the hosting arrangement or "us-west-2" remains a valid braket region. https://docs.aws.amazon.com/braket/latest/developerguide/braket-devices.html

Reply:
But how are we testing that they are filtered out?

Reply:
The anealers don't have the `DeviceActionType.JAQCD` action, so we won't be able to get the `supportedOperations` from these devices to construct `BackendInfo`s. If the filter isn't working this test would throw an exception. But I suppose I can improve the test to further make sure they are not in the returned list.

Reply:
OK I see, that's fine.